client/src/views/Overview.svelte

```diff
@@ -17,10 +18,30 @@
 	{/each}
 </div>
 
+
 <style>
 	.grid {
 		display: grid;
 		grid-template-columns: repeat(auto-fit, minmax(320px, 1fr));
 		justify-content: space-between;
 	}
+	#slideshow{
+		flex-direction: row;
+font-size: 2em;
+	}
+	#right{
+
+		float:right;
```

Comment:
Eigentlich wann immer du mit `float` layoutest ist irgendwas ganz gewaltig schief gelaufen, aber ich nehm's keinem Informatikstudent übel, jetzt kein CSS flexbox perfektioniert zu haben :P


... ich glaube das CSS ist eh unused?

Reply:
Das hab ich schon lange gesucht :) War mit float nie zufrieden, hab das aber mal so gelernt.

Reply:
Tatsächlich hat unser seit-15-jahren-nicht-in-der-Industrie-Prof im Bachelor und auch float layouts gezeigt. Und schon hat jeder CSS gehasst und wollte nie mehr web development sehen.

Dabei gibts inzwischen `flex` und `grid` und die float layouts waren schon immer ein scheiss Hack, um _überhaupt_ irgendwie layouten zu können als es noch nix anderes gab. Jo, kann man sich mal anschauen – vor allem flex zu beherrschen lohnt sich aber extrem wenn man CSS macht. Ich bin inzwischen so weit, dass ich will dass flex der default Style von jedem div ist 😂